test/test_cfparser.ini

```diff
@@ -12,3 +12,4 @@ str2 = val2
 
 time1 = 1.5
 time2 = 2.5
+'test key' = 'test value'
```

Comment:
> Is it possible to enter a keyname containing a newline? Databasenames can sadly contain all kinds of characters. It would be good to support that.

This doc below implies that anything is acceptable as a name if quoted:

https://www.postgresql.org/docs/7.0/syntax525.htm#:~:text=Names%20containing%20other%20characters%20may%20be%20formed%20by%20surrounding%20them%20with%20double%20quotes%20(%22).%20For%20example%2C%20table%20or%20column%20names%20may%20contain%20otherwise%20disallowed%20characters%20such%20as%20spaces%2C%20ampersands%2C%20etc.%20if%20quoted.